docs/guides/integration-mobile-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+title: UID2 Mobile Integration Overview for Android and iOS
+sidebar_label: Integration Overview for Mobile
+pagination_label: Mobile Integration Overview for Android and iOS
+description: Overview of options for UID2  mobile integration.
+hide_table_of_contents: false
+sidebar_position: 04
+displayed_sidebar: sidebarPublishers
+---
+
+import Link from '@docusaurus/Link';
+
+# UID2 Mobile Integration Overview for Android and iOS
+
+This guide is an overview of integration options for mobile app publishers who want to integrate with UID2 using the UID2 SDK for Android or the UID2 SDK for iOS.
```

Comment:
this sentence is cloned based on starting sentence from other existing overview pages. But is this really a _guide_? or

_This is an overview of integration options ...._

sufficient?

Reply:
Not changing for now. I definitely see the point but can't think of a better way to word it.